eclass/ecm.eclass

```diff
@@ -694,6 +727,14 @@ ecm_src_install() {
 			popd > /dev/null || die
 		fi
 	fi
+
+	for f in "${ECM_REMOVE_FROM_INSTALL[@]}"; do
+		if [[ -d ${f} ]]; then
+			rm -r "${ED}"${f} || die
+		else
+			rm "${ED}"${f} || die
+		fi
+	done
```

Comment:
`[[ -d ]]` check feels like unnecessary complexity, could -r in either case assuming I'm not missing something.

Given `${f}` is unquoted, I assume that supporting globs in `${f}` is wanted, right? *If* didn't want that, could remove the loop too, e.g.:
```bash
	if (( ${#ECM_REMOVE_FROM_INSTALL[@]} )); then
		rm -r "${ECM_REMOVE_FROM_INSTALL[@]/#/${ED}}" || die
	fi
```
But globs sound like they could be useful, so just saying.

Reply:
(albeit fwiw globs may be error prone if someone forgets to quote them and it pickup system files inside the array)

Reply:
@sam wdyt? If in doubt probably restrict globbing, otoh this eclass will see very specialised use...

Reply:
I'm inclined to say we allow globbing given the already interesting use of the escaped vars we pass in..